mir_eval/util.py

```diff
@@ -621,9 +621,120 @@ def match_events(ref, est, window):
     # Construct the graph input
     G = {}
     for ref_i, est_i in zip(*hits):
-        if ref_i not in G:
-            G[ref_i] = []
-        G[ref_i].append(est_i)
+        if est_i not in G:
+            G[est_i] = []
+        G[est_i].append(ref_i)
+
+    # Compute the maximum matching
+    matching = sorted(_bipartite_match(G).items())
+
+    return matching
+
+
+def match_notes(ref_intervals, ref_pitches, est_intervals, est_pitches,
```

Comment:
Are you proposing to move all the code in `match_notes` into `precision_recall_f1` (inline)? At the moment it's true that `match_notes` is only used by `precision_recall_f1`, not sure if any future metrics would make use of it. I do however like the separation just because it makes both methods more readable (matching code in one place, metrics derived from matching in another place). Anyway, if you think it's better I can move the code into `precision_recall_f1` and remove `match_notes` altogether. Or, how about I move `match_notes` into transcription.py, but keep it as a separate method?